fixit/rules/use_assert_in.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.helpers import ensure_type
+
+from fixit.common.base import CstLintRule
+from fixit.common.utils import InvalidTestCase as Invalid, ValidTestCase as Valid
+
+
+class UseAssertInRule(CstLintRule):
+    """
+    Discourages use of ``assertTrue(x in y)`` and ``assertFalse(x in y)`` as it is deprecated (https://docs.python.org/3.8/library/unittest.html#deprecated-aliases).
+    Use ``assertIn(x, y)`` and ``assertNotIn(x, y)``) instead.
+    """
+
+    ONCALL_SHORTNAME = "instagram_server_framework"
+    MESSAGE = "Use assertIn/assertNotIn instead of assertTrue/assertFalse for inclusion check. (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn)"
+
+    VALID = [
+        Valid("self.assertIn(a, b)"),
+        Valid("self.assertIn(f(), b)"),
+        Valid("self.assertIn(f(x), b)"),
+        Valid("self.assertIn(f(g(x)), b)"),
+        Valid("self.assertNotIn(a, b)"),
+        Valid("self.assertNotIn(f(), b)"),
+        Valid("self.assertNotIn(f(x), b)"),
+        Valid("self.assertNotIn(f(g(x)), b)"),
+    ]
+
+    INVALID = [
+        Invalid(
+            "self.assertTrue(a in b)",
+            expected_replacement="self.assertIn(a, b)",
+        ),
+        Invalid(
+            "self.assertTrue(f() in b)",
+            expected_replacement="self.assertIn(f(), b)",
+        ),
+        Invalid(
+            "self.assertTrue(f(x) in b)",
+            expected_replacement="self.assertIn(f(x), b)",
+        ),
+        Invalid(
+            "self.assertTrue(f(g(x)) in b)",
+            expected_replacement="self.assertIn(f(g(x)), b)",
+        ),
+        Invalid(
+            "self.assertTrue(a not in b)",
+            expected_replacement="self.assertNotIn(a, b)",
+        ),
+        Invalid(
+            "self.assertTrue(not a in b)",
+            expected_replacement="self.assertNotIn(a, b)",
+        ),
+        Invalid(
+            "self.assertFalse(a in b)",
+            expected_replacement="self.assertNotIn(a, b)",
+        ),
+    ]
+
+    def visit_Call(self, node: cst.Call) -> None:
+        if m.matches(
+            node,
+            m.Call(
+                func=m.Attribute(value=m.Name("self"), attr=m.Name("assertTrue")),
```

Comment:
may as well find `assertFalse` too

Reply:
makes sense.
this was taken from the internal rule though. i'll leave it for now so there's consistency between both places.

Reply:
Yeah, use a single extract can be more effective and that can leave as future work. @dkteo can you add a `# TODO: ` comment to the code as a note?
Also, once this is merged, we'll want to remove the internal copy to avoid duplication since this will be released to internal dev environment.